api/swagger.yaml

```diff
@@ -1195,6 +1195,7 @@ definitions:
               - "default"
               - "process"
               - "hyperv"
+              - ""
```

Comment:
Wondering if we should try if some permutation of  "nullable" would work for this; it looks like it's hard to do correct in combination with enum though; https://github.com/OAI/OpenAPI-Specification/issues/1900

Reply:
Thanks! Yes, I also went looking, and it looks like options are a bit limited for Swagger / OpenAPI 2 sadly. We do have the `x-nullable: true` option, but I don't think that helps for this case.

Unfortunately, we're currently stuck at OpenAPI 2 (due to some go frameworks we use for these), so I think this is indeed the best we can do for now.